metricq_tools/summary.py

```diff
@@ -0,0 +1,294 @@
+                "Distribution of the duration between consecutive data points in seconds",
+                fg="yellow",
+            )
+        )
+        click.echo()
+
+        self._print_histogram(intervals)
+
+        click.echo()
+        click.echo()
+
+    def _print_values_histogram(self, values):
+        click.echo(
+            click.style("Distribution of the values of the data points", fg="yellow")
+        )
+        click.echo()
+
+        self._print_histogram(values)
+
+        click.echo()
+        click.echo()
+
+    def print(self):
+        for metric in self._metrics:
+
+            if self.values[metric]:
+                click.echo()
+                click.echo(click.style(f"Stats of metric {metric!r}:", fg="green"))
+                click.echo()
+
+                if self.print_intervals and self.last_timestamp[metric]:
+                    self._print_intervals_histogram(self.intervals[metric])
+
+                if self.print_values:
+                    self._print_values_histogram(self.values[metric])
+            else:
+                click.echo()
+                click.echo(
+                    click.style(f"No Data for metric {metric!r} received!", fg="red")
+                )
+                click.echo()
+
+        if self.print_stats:
+            self._print_statistics()
+
+    def _print_statistics(self):
+        click.echo(
+            click.style(
+                "Statistics",
+                fg="yellow",
+            )
+        )
+        click.echo()
+
+        table = list[list]()
+        table.append(["Minimum"])
+        table.append(["Maximum"])
+        table.append(["Average"])
+        table.append(["Median"])
+        table.append(["Standard deviation"])
+        table.append(["Arithmetic mean"])
+        table.append(["Variance"])
+        table.append(["Count"])
+
+        for metric in self._metrics:
+            if self.values[metric]:
+                table[0].append(np.amin(self.values[metric]))
+                table[1].append(np.amax(self.values[metric]))
+                table[2].append(np.average(self.values[metric]))
+                table[3].append(np.median(self.values[metric]))
+                table[4].append(np.std(self.values[metric]))
+                table[5].append(np.mean(self.values[metric]))
+                table[6].append(np.var(self.values[metric]))
+                table[7].append(len(self.values[metric]))
+            else:
+                table[0].append("n/a")
+                table[1].append("n/a")
+                table[2].append("n/a")
+                table[3].append("n/a")
+                table[4].append("n/a")
+                table[5].append("n/a")
+                table[6].append("n/a")
+                table[7].append(len(self.values[metric]))
+
+        click.echo(tabulate(table, headers=self._metrics, tablefmt="fancy_grid"))
+
+        click.echo()
+        click.echo()
+
+
+async def run_cmd(command):
+    click.echo(f"running... {command!r}")
```

Comment:
This should IMO be a `DEBUG`-level log message:

```suggestion
    logger.debug("Running command: {!r}", command)
```